pkg/netceptor/netceptor.go

```diff
@@ -730,6 +734,15 @@ func (s *Netceptor) GetServerTLSConfig(name string) (*tls.Config, error) {
 	return sc.Clone(), nil
 }
 
+// AddWorkCommand records a work command so it can be included in service announcements
+func (s *Netceptor) AddWorkCommand(command string) error {
+	if command == "" {
```

Comment:
One thing I'm unclear on @fosterseth is if we need a mutex around this; I'm unsure if the registration of the work commands in `RegisterWorker` happens in competing goroutines.  I see other fields on this struct in this file seem to follow the pattern of locking access to a mutex, so I may need to do so here as well for safety.

Reply:
Yeah I don't know enough about receptor internals anymore to know either

Reply:
I think this is safe. RegisterWorker is called one at a time on each `work-command` in the yaml.

Reply:
Well, I wrapped it w/ a service ad lock just to be sure.